buyer-signing-service/src/server.js

```diff
@@ -2,11 +2,14 @@ import 'babel-polyfill';
 import app from './app';
 import config from '../config';
 import logger from './utils/logger';
+import writePid from './utils/writePid';
 
 const server = () => {
   const { port, host, env } = config;
-  app.listen({ port, host }, () =>
-    logger.info(`Buyer Signing Service listening on port ${port} and host ${host} in ${env} mode`));
+  app.listen({ port, host }, () => {
+    logger.info(`Buyer Signing Service listening on port ${port} and host ${host} in ${env} mode`);
+    writePid(`${__dirname}/../service.pid`);
```

Comment:
Why is this necessary?

Reply:
I'm not sure why is there at all or when we did it. It is handy, though.